### DISCUSSION TITLE
Real box in Fortran

### DISCUSSION THREAD

**Discussion by Mephiiistopheles** [AMReX-Codes/amrex#3250] [General]
Hi all,

I've been following the Multifab tutorial and trying to recreate it using the Fortran interface. Am I missing something, or is there not currently any provision for directly creating a Real Box via the Fortran interface, such that specifying the physical dimensions of the domain is most readily done via an input file?

Many thanks in advance.

**Comment by WeiqunZhang** [comment]
There is no RealBox in Fortran. But you can call `amrex_geometry_set_prob_domain` instead of using an input file to specify the physical domain size. This needs to be done before any Geometry objects are created.

Note that traditionally all amrex::Geometry objects had the same problem domain. We relaxed that constraint and now in C++ one can have different problem domains in different Geometry objects. However in Fortran, we still have all amrex_geometry objects sharing the same problem domain.

**Reply by Mephiiistopheles** [reply]
That works perfectly. Thanks a lot for your help.
